### ISSUE TITLE
VPA recommender constantly crashing on AKS 1.25.5

### ISSUE THREAD

**Issue by tberreis** [reactiveops/charts#1098]
### What happened?

When using VPA with helm chart 1.6.1, app version 0.11.0 on AKS 1.24.6 everything works as expected.
But after upgrading AKS to 1.25.5 without any other changes, VPA begins to crash constantly with the following events:

```
Liveness probe failed: HTTP probe failed with statuscode: 500
Readiness probe failed: HTTP probe failed with statuscode: 500
```

Am I doing something wrong or is this a known issue?
Not sure if 1.25.5 is included in the supported version range "1.23-1.25" - see https://github.com/FairwindsOps/charts/pull/1086.

As documented in https://github.com/kubernetes/autoscaler/tree/master/vertical-pod-autoscaler#installation the image version 0.11.0 does not support AKS 1.25.

### What did you expect to happen?

VPA running smoothly as on AKS 1.24.6

### How can we reproduce this?

- Create a cluster with AKS version 1.24.6
- Deploy VPA and check whether it is running
- Upgrade AKS to 1.25.5 and check for pod restarts after around 10 mins
![image](https://user-images.githubusercontent.com/88875030/220409059-e7b24d9f-cf98-479f-a516-e8b46f671717.png)

- Patch image version to k8s.gcr.io/autoscaling/vpa-recommender:**0.13.0** and check again after some minutes

### Version

vpa 1.6.1

### Search

- [X] I did search for other open and closed issues before opening this.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Additional context

PS: I know that it's possible to overwrite the image tag via .Values.recommender.image.tag (https://github.com/FairwindsOps/charts/blob/master/stable/vpa/templates/recommender-deployment.yaml#L42) but that's not the preferred solution.

**Comment by djsly** [comment]
came to report the same issue. sounds like a simple image bump in the chart ?

**Comment by PDVJAM** [comment]
Same here.

**Comment by dmitrii-branitskii** [comment]
[check other repo](https://cowboysysop.github.io/charts/)

**Comment by sudermanjr** [comment]
It seems like this should be resolved with the update to v0.13.0 and chart >=1.7.0
